sample/src/main/res/layout/activity_main.xml

```diff
@@ -22,6 +22,7 @@
         app:dayBackgroundColor="#05000000"
         app:todayBackgroundColor="#1848adff"
         app:headerColumnBackground="#ffffffff"
-        app:todayHeaderTextColor="@color/accent"/>
+        app:todayHeaderTextColor="@color/accent"
+        app:showHalfHours="false"/>
```

Comment:
Hi, i just have tried your code, but if i change my MainActivity code using your code, mWeekView.setDateTimeInterpreter(new DateTimeInterpreter() is redflagged. Can you notice me where i wrong? Thanks